frontend/packages/dev-console/src/components/pipelines/modals/common/PipelineSecretSection.scss

```
@@ -1,12 +1,14 @@
 .odc-pipeline-secret-section {
   margin-top: var(--pf-global--spacer--sm);
   padding-left: var(--pf-global--spacer--lg);
+  &__secrets {
+    padding: var(--pf-global--spacer--md);
```

I think you just want the indent... spacing top causes an odd gap, spacing bottom is unnecessary because of already the large spacing applied... and spacing right is probably not desirable... it makes the dotted-outline section not align with the edge of the dropdowns / fields in the form.
```suggestion
    padding-left: var(--pf-global--spacer--md);
```